From 2d9612a4e650cee71d1c34f0b7d2eeaae876cdf3 Mon Sep 17 00:00:00 2001
From: qianmenx <qianx.meng@intel.com>
Date: Fri, 17 May 2019 11:31:05 +0800
Subject: [PATCH 2334/2345] media: intel-ipu4: ox03a10: delete unused exposure
 mode

Message for Open Source:
delete unused exposure mode(x3a)

Change-Id: Id49c9255026245d7b6df48d5aeab3c4d177b23eb
Signed-off-by: qianmenx <qianx.meng@intel.com>
(cherry picked from commit 1cf9a58623598b5ef6036749ea5c954b129e1c0f)
Signed-off-by: lijuncax <lijunx.cai@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_common.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
index 067bae6..486c9ca 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
@@ -1016,10 +1016,6 @@ struct crl_ctrl_data_pair ox03a10_ctrl_data_modes[] = {
 	{
 		.ctrl_id = CRL_CID_EXPOSURE_MODE,
 		.data = 1,
-	},
-	{
-		.ctrl_id = CRL_CID_EXPOSURE_MODE,
-		.data = 2,
 	}
 };
 
-- 
2.7.4

